proxy/m2_actions.c

```diff
@@ -1099,6 +1119,8 @@ action_m2_container_destroy (struct req_args_s *args)
 			err = m2v2_remote_execute_DESTROY_many(urlv+1, args->url,
 					flag_force);
 		}
+
+		// free(url);
```

Comment:
Should that commented code be removed ?
If not, `g_free` would be more suitable :)